pkg/apis/appstacks/v1beta1/runtimecomponent_types.go

```diff
@@ -68,6 +68,8 @@ type RuntimeComponentService struct {
 	// +kubebuilder:validation:Minimum=1
 	Port int32 `json:"port,omitempty"`
 
+	Name string `json:"name,omitempty"`
```

Comment:
`Name` might be interpreted as name of the service. `portName` would be more descriptive and avoid ambiguity.